app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,93 @@
+<li class="tag">Programming and debugging</li>
+<li class="tag">Creative projects</li>
+<li class="tag">Use of logical reasoning</li>
+<li class="tag">Computer systems</li>
+<li class="tag">Uses of technology</li>
+<li class="tag">Online safety </li>
+</ul>
+<p>You'll teach 14 to 16 year olds (key stage 4) computer science as a GCSE qualification, teaching pupil's ways to protect their online privacy and identity. Themes you might also cover are:</p>
+<ul class="vertical-tags">
+<li class="tag">Computational thinking</li>
+<li class="tag">Digital media and information technology</li>
+<li class="tag">Computer networks</li>
+<li class="tag">Cyber security</li>
+<li class="tag">Data representation</li>
+<li class="tag">Digital technology</li>
+</ul>
+<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as an A level qualification, helping pupils to develop websites and prepare them for further learning or employment. Themes you might also cover are:</p>
+<ul class="vertical-tags">
+<li class="tag">Abstraction</li>
+<li class="tag">Writing programmes to solve problems</li>
+<li class="tag">Efficiency of algorithms</li>
+<li class="tag">Data structures and the use of databases</li>
+<li class="tag">Systems architecture</li>
+<li class="tag">Computing related mathematics</li>
+</ul>
+
+<p><a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks4-l/units">Explore what teaching a computer lesson would be like</a>.</p>
+</section>
+<section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
+    name: "Shiulee Begum, Curriculum Lead Computing and Business",
+    large: true
+  ) %>
+</section>
+
+  <section class="col col-720">
+    <h2>Check your qualifications</h2>
+<p>To train to teach in primary and secondary schools in England, you’ll need:</p>
+<ul>
+<li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
+<li>a bachelor’s degree in any subject</li>
+</ul>
+<p>Computer science teacher training courses have had successful applications from candidates with a range of degrees, including business and management, data science, history and economics.</p>
+<p><a href="/non-uk-teachers/train-to-teach-in-england-as-an-international-student">Non-UK citizens can find out more about training to teach in England</a>.</p>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a computing teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge.</p>
```

Comment:
```suggestion
<p>If your training provider thinks you need to top up your computing knowledge, you might need to do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training.</p>
```

I think the way this is worded in production at the moment is a bit misleading - it's not really before you become a teacher, it's before you start your training. I've suggested rewording based on the SKE page - please tweak as needed.